Dockerfile

```diff
@@ -4,6 +4,7 @@ ENV TERM xterm-256color
 
 RUN apt-get update && \
     apt-get --fix-missing -y --force-yes --no-install-recommends install git ca-certificates && \
+    rm -rf /pljava && \
```

Comment:
So this directory should not exist before the following line. How is it you're hitting this issue if we haven't yet clone the repo yet? Maybe a stale docker cache or you're doing things in a non-standard way?

Reply:
I haven't got a clue. It could be stale docker cache. Not sure how to verify that. Nevertheless, this could be a good fail-save. If it happened to my build, it could happen to someone else's build.

Reply:
If you don't mind can you provide the build command you are using? You should be able to use the `--no-cache` option to skip docker using any cache and get around your problem.

The change in question is safe just should be unnecessary is all.

Reply:
`docker build -t pega-database:8.3.0-225 .`
I'll try running it with the --no-cache option.

Reply:
That is flipping weird and shouldn't be happening. I can't really explain how things could pop if the folder does not exist yet. The only thing that makes sense is something in the docker cache.

Reply:
I agree. It shouldn't be happening. That folder shouldn't have been there in the first place. But I didn't create it manually. So it seems that it could be something that occurs (maybe even randomly) on machines. It's definitely weird.

Reply:
I'll leave this PR open for a little while longer to see if we magically stumble upon something but I'm under the current assumption that this shouldn't be needed.